app/konnect/dev-portal/customization/netlify.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Self-Host your Dev Portal with Netlify
+content_type: how-to
+badge: oss
+---
+
+You can use the self-hosted, open source Dev Portal to display your APIs to developers on a self-hosted website. This guide shows you how to configure an example open source Dev Portal with Netlify, which you can further customize by adjusting the code of the sample app and self-hosting it on a frontend application of your choice.
+
+The example application is pre-configured with the following features:
+* All [features in the {{site.konnect_short_name}}-hosted Dev Portal](/konnect/dev-portal/customization/)
+* Localization is set to English by default. You can configure it using [`src/locales`](https://github.com/Kong/konnect-portal/tree/main/src/locales).
+* [UI test automation through Cypress](https://github.com/Kong/konnect-portal/tree/main/cypress)
+
+## Prerequisites
+
+* [A {{site.konnect_saas}} account](/konnect/getting-started/access-account/)
+* [Yarn 1.22.x installed](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* [A Netlify account](https://www.netlify.com/)
+
+## Configure your self-hosted domains in {{site.konnect_short_name}}
+
+1. In {{site.konnect_short_name}}, click {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings) in the sidebar.
+1. Click the **Portal Domain** tab.
+1. Enter a name for your custom portal domain API in the **Custom Hosted Domain** field. For example, `api.mycompany.com`. 
+1. Enter a name for your custom portal client domain in the **Custom Self-Hosted UI Domain** field. For example, `client.mycompany.com`.
+1. After you configure these domains in {{site.konnect_short_name}}, add a CNAME record to your domain's DNS records that points the custom API domain to the default portal domain. For example:
+    ```sh
+    api.mycompany.com   CNAME   3600    client.mycompany.com
+    ```
+
+Now the self-hosted Dev Portal is enabled and the {{site.konnect_short_name}}-hosted Dev Portal is disabled. The custom portal domain API you specified will be used as the API domain of your portal and the custom portal client domain will be used to access the UI of your self-hosted portal.
+
+## Deploy your self-hosted Dev Portal with Netlify
+
+In this example, you will deploy the self-hosted example Dev Portal using Netlify. You can use any static site generator of your choice to host your Dev Portal.
+
+1. In GitHub, [fork a copy](https://docs.github.com/get-started/quickstart/fork-a-repo) of the [open source Dev Portal application example](https://github.com/Kong/konnect-portal).
```

Comment:
```suggestion
1. In GitHub, [fork](https://docs.github.com/get-started/quickstart/fork-a-repo) the [open source Dev Portal application example](https://github.com/Kong/konnect-portal).
```
Small nitpick - not forking a copy, you're forking the repo, as the fork is the copy.